python/pyspark/sql/pandas/types.py

```diff
@@ -86,8 +86,13 @@ def to_arrow_type(dt: DataType) -> "pa.DataType":
     elif type(dt) == DayTimeIntervalType:
         arrow_type = pa.duration("us")
     elif type(dt) == ArrayType:
-        if type(dt.elementType) in [StructType, TimestampType]:
+        if type(dt.elementType) == TimestampType:
```

Comment:
My only concern here is that we current convert the struct type as a DataFrame instead a Series of dictionaries (e.g., `returnType=struct<...>`) which is inconsistent with the current conversion this PR proposes (a Series of dictionaries).

Reply:
Array of StructType now follows the same type of serialization as StructType.

Reply:
Array of StructType now follows the same type of serialization as StructType.